### ISSUE TITLE
Per-pod persistent variables

### ISSUE THREAD

**Issue by Alexustas** [Mihara/RasterPropMonitor#232]
I've tried to use the new feature "Per-pod persistent variables" and faced the problem. Immediately after the start this variable gives its own name, instead of "0" or "1", untill  the appropriate switch will be pressed for the first time. After that,  all showing okay .

I would like to ask you to change this if given persistent  not found, (switcher with such persistent is not installed in the cockpit) the variable displays the status of this persistent would be equal to "-1" (now it displays its own name)

 And also, is it possible to make a switcher enabled by default at the start?

**Comment by MOARdV** [comment]
Re: mapped variable not found - yes, I can change it to -1 instead of returning its name.  I will also see if I can find a way to store an initial value so that the variable exists.

Re: switcher enabled by default - I think that is Issue #119 .  I can work on implementing that.

**Comment by MOARdV** [comment]
Okay.  I've updated the PERSISTENT_ behavior to return -1 if the persistent variable is missing.  I have also updated JSIActionGroupSwitch to save its persistent value at startup, so if the switch is in the pod, it should not return -1.  I *also* added `initialState` to actionGroupSwitch.  This will only work with dummy and intlights persistent values - action groups and plugin switches are always set to their initial state by the action group or plugin.  I have updated the [RPM dev build](https://www.dropbox.com/s/n02079q3rz9makd/RasterPropMonitor-dev.0.19.5.zip?dl=0).
